devtools/scumm-md5.txt

```diff
@@ -489,7 +489,7 @@ freddi	Freddi Fish 1: The Case of the Missing Kelp Seeds
 	cf8ef3a1fb483c5c4b1c584d1167b2c4	-1	de	Windows	HE 73	-	-	Oncer
 	1f2e62b5a9c50589fc342285a6bb3a27	-1	he	Windows	HE 73	-	-	e_orz
 	507bb360688dc4180fdf0d7597352a69	26402	se	Windows	HE 73	-	-	Sven Arvidsson
-	df047cc4792150f601290357566d36a6	-1	us	All	HE 90	Updated	-	khalek
+	df047cc4792150f601290357566d36a6	-1	en	All	HE 90	Updated	-	khalek
```

Comment:
I do not have both (unfortunately), and even then it's possible that multiple versions exists of the same language.

My assumption is that for these type of games MD5 clashes are nearly impossible. There is no registered clash yet. But if there actually is a clash, then there is a higher probability of it being reported if we report incorrect information, instead of generic. If we get a report I will gladly try to detect both versions properly.